scona/visualisations.py

```diff
@@ -0,0 +1,352 @@
+import warnings
+
+import numpy as np
+import networkx as nx
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from scona.visualisations_helpers import save_fig
+from scona.visualisations_helpers import df_sns_barplot
+
+
+def plot_rich_club(brain_bundle, original_network, figure_name=None, color=None,
+                   show_legend=True, x_max=None, y_max=None):
+    """
+    This is a visualisation tool for plotting the rich club values per degree
+    along with the random rich club values created from a random networks
+    with a preserved degree distribution.
+
+    Parameters
+    ----------
+    brain_bundle : `GraphBundle` object
+        a python dictionary with BrainNetwork objects as values
+        (:class:`str`: :class:`BrainNetwork` pairs), contains original Graph
+        and random graphs.
+    original_network: str, required
+        This should index the particular network in `brain_bundle` that you want
+        the figure to highlight. A distribution of all the other networks in
+        `brain_bundle` will be rendered for comparison.
+        To plot rich club values along with the rich club values from
+        randomed graphs it is required to pass the name of the initial network,
+        the proper network, the one you got from the mri data.
```

Comment:
I would like to remove the following from all function descriptions:
```
       To plot rich club values along with the rich club values from
        randomed graphs it is required to pass the name of the initial network,
        the proper network, the one you got from the mri data.
```

Reply:
Removed